lms/static/js/jquery.gradebook.js

```diff
@@ -57,16 +58,23 @@ var Gradebook = function($element) {
 	};
 
 	var updateWidths = function(e) {
+		// Here updating direction of scroll.
```

Comment:
Same issue here as above. Comments should explain what the code is *currently* doing, not explain changes.